readthedocs/oauth/admin.py

```diff
@@ -16,7 +16,42 @@ class RemoteRepositoryAdmin(admin.ModelAdmin):
 
     """Admin configuration for the RemoteRepository model."""
 
+    readonly_fields = ('created', 'modified',)
     raw_id_fields = ('organization',)
+    list_select_related = ('organization',)
+    list_filter = ('vcs_provider', 'vcs', 'private',)
+    search_fields = (
+        'name',
+        'full_name',
+        'html_url',
```

Comment:
`html_url` is probably not needed. I think in most cases will be 80% equal to `full_name`, right?

Reply:
Yup, We can remove it! :+1: